test-runner.js

```diff
@@ -0,0 +1,5 @@
+const testsContext = require.context('./tests', true, /\.js$/);
+const componentsContext = require.context('./src/', true, /components\/.*\.js$/);
```

Comment:
Because server tests can't be run via karma since they're nothing to do with the DOM.